curriculum/challenges/english/25-front-end-development/workshop-mathbot/66ea845f54195c7020cc521b.md

```diff
@@ -0,0 +1,76 @@
+---
+id: 66ea845f54195c7020cc521b
+title: Step 5
+challengeType: 1
+dashedName: step-5
+---
+
+# --description--
+
+The formula to generate a random number between two values is the following:
+
+```js
+Math.random() * (max - min) + min;
```

Comment:
You could change this example to use maybe `maximum` and `minimum` or `maxNumber` and `minNumber` or something - so people can't just copy/paste the example.